Week1/homework/world-queries.txt

```diff
@@ -37,4 +39,199 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
++-----------+------------+
+| Name      | Population |
++-----------+------------+
+| Peking    |    7472000 |
+| Pusan     |    3804522 |
+| Pyongyang |    2484000 |
++-----------+------------+
+3 rows in set (0.00 sec)
+
+7. What is the population in Asia?
+
+SELECT Continent,SUM(Population) FROM country 
+WHERE country.Continent = 'Asia';
++-----------+-----------------+
+| Continent | SUM(Population) |
++-----------+-----------------+
+| Asia      |      3705025700 |
++-----------+-----------------+
+1 row in set (0.00 sec)
+
+8. Which languages are spoken in the Region of ‘South America’
+
+SELECT country.Name,Language FROM countrylanguage 
+JOIN country ON country.Code = countrylanguage.Countrycode 
+WHERE country.Region = 'South America';
++------------------+------------------+
+| Name             | Language         |
++------------------+------------------+
+| Argentina        | Indian Languages |
+| Argentina        | Italian          |
+| Argentina        | Spanish          |
+| Bolivia          | Aimará           |
+| Bolivia          | Guaraní          |
+| Bolivia          | Ketšua           |
+| Bolivia          | Spanish          |
+| Brazil           | German           |
+| Brazil           | Indian Languages |
+| Brazil           | Italian          |
+| Brazil           | Japanese         |
+| Brazil           | Portuguese       |
+| Chile            | Aimará           |
+| Chile            | Araucan          |
+| Chile            | Rapa nui         |
+| Chile            | Spanish          |
+| Colombia         | Arawakan         |
+| Colombia         | Caribbean        |
+| Colombia         | Chibcha          |
+| Colombia         | Creole English   |
+| Colombia         | Spanish          |
+| Ecuador          | Ketšua           |
+| Ecuador          | Spanish          |
+| Falkland Islands | English          |
+| French Guiana    | Creole French    |
+| French Guiana    | Indian Languages |
+| Guyana           | Arawakan         |
+| Guyana           | Caribbean        |
+| Guyana           | Creole English   |
+| Peru             | Aimará           |
+| Peru             | Ketšua           |
+| Peru             | Spanish          |
+| Paraguay         | German           |
+| Paraguay         | Guaraní          |
+| Paraguay         | Portuguese       |
+| Paraguay         | Spanish          |
+| Suriname         | Hindi            |
+| Suriname         | Sranantonga      |
+| Uruguay          | Spanish          |
+| Venezuela        | Goajiro          |
+| Venezuela        | Spanish          |
+| Venezuela        | Warrau           |
++------------------+------------------+
+42 rows in set (0.00 sec)
+
+9. What are the languages spoken on all cities named ‘Barcelona’ (you may need to join 3 tables =) )
+
+SELECT city.Name,country.Name,Language FROM countrylanguage 
+JOIN city ON city.Countrycode = countrylanguage.Countrycode 
+JOIN country ON city.Countrycode = country.Code 
+WHERE city.Name = 'Barcelona';
```

Comment:
👍

Wow, when I was testing to check the correct answers, I did the query in the exact same way as you, except I didn't include `country.Name` in the results - I assumed all seven languages where from Barcelona, ESP. I had no idea some of them were from Venezuela!